### PR TITLE
harmonize colors on dashboard

### PR DESCRIPTION
**Pull request by martinvibes**
hello @djeck1432  @lukaspetrasek please sorry for the PR i closed my vscode was having some minor issues so i used codespaces instead 
Please kindly review :)

### CODE REVIEW

**Comment by lukaspetrasek** [comment]
Hi @martinvibes , there are conflicts in your PR, pls solve them. It looks like your master is behind 😉

**Comment by martinvibes** [comment]
hello @lukaspetrasek @djeck1432 kindly review :)

**Comment by djeck1432** [comment]
@martinvibes Hi, I just checked your PR, you removed a lot of code, please, do rebase again

**Comment by martinvibes** [comment]
hello @djeck1432 i have rebase my code and both branches are up to date

File: app.py [hunk 1]
```diff
@@ -13,70 +15,136 @@
...
 import src.swap_amm
-import src.utils
-from src.chart_utils import (
-    get_protocol_data_mappings,
-    load_stats_data,
-    transform_loans_data,
-    transform_main_chart_data,
-)
 
-PROTOCOL_NAMES = [
```
Comment (djeck1432) [inline app.py:24]: This should be removed

File: app.py [hunk 2]
```diff
@@ -13,70 +15,136 @@
...
-def create_stablecoin_bundle(
-    data: dict[str, pandas.DataFrame]
-) -> dict[str, pandas.DataFrame]:
+def _get_available_liquidity(data: pandas.DataFrame, price: float, price_diff: float, bids: bool) -> float:
+    price_lower_bound = max(0.95 * price, price - price_diff) if bids else price
+    price_upper_bound = price if bids else min(1.05 * price, price + price_diff)
+    return data.loc[data["price"].between(price_lower_bound, price_upper_bound), "quantity"].sum()
+
+
+def add_ekubo_liquidity(
```
Comment (djeck1432) [inline app.py:47]: This function is not more in this file, this one should be removed

**Review by djeck1432 — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (martinvibes) [reply app.py:24]: hello @djeck1432 this requested change is not in my file, but it is currently in your file. Or do you mean i should add it to my file?

> This should be removed

Comment (djeck1432) [reply app.py:24]: But in your code, you removed `PROTOCOL_NAMES`

**Comment by martinvibes** [comment]
hello @djeck1432 i have removed the function

**Comment by djeck1432** [comment]
@martinvibes Please, you should do it again, your rebase has a lot of wrong changes. Let's jump on call and do it together.

**Pull request closed without merging**